features/shipping/managing_shipments/filtering_shipments_by_state.feature

```diff
@@ -0,0 +1,37 @@
+@managing_shipments
+Feature: Filtering shipments from multiple channels by state
```

Comment:
I think we don't need multiple channels in this feature file to show that `state` filter works. This would decrease the `Background` section noticeably 😄